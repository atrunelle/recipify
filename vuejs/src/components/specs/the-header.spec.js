import { mount } from '@vue/test-utils';
import TheHeader from '../the-header';

describe('Component: the header', () => {
  let component;

  beforeEach(() => {
    component = mount(TheHeader, {
      stubs: {
        'the-menu': '<nav></nav>',
      },
    });
  });

  it('should match snapshot', () => {
    expect(component.html()).toMatchSnapshot();
  });
});
