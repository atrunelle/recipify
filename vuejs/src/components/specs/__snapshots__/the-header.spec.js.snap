// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: the header should match snapshot 1`] = `
<header>
    <nav></nav>
    <div class="jumbotron" style="height: 400px;">
        <div class="jumbotron__wrapper">
            <div class="jumbotron__background light-green"></div>
            <div class="jumbotron__content">
                <div class="container fill-height">
                    <div class="layout column align-center justify-center">
                        <img src="/static/logo.svg" alt="logo" class="the-header__logo">
                        <h1>Recipify</h1>
                    </div>
                </div>
            </div>
        </div>
    </div>
</header>
`;
